index.js

```diff
@@ -1,29 +1,11 @@
+const EventEmitter = require('events');
```

Comment:
@rwjblue I've added `'use strict';` and 0.12 started to fail. 
What's should we do next? revert `use strict` or update test matrix?